scripts/update_client_permissions.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import logging
+import argparse
+
+logger = logging.getLogger(__name__)
+
+def login_admin(admin_id, admin_secret, url):
+    headers = {'Content-Type': 'application/x-www-form-urlencoded',
+               'Accept': 'application/json'}
+    payload = {'grant_type': 'client_credentials',
+               'response_type': 'token',
+               'token_format': 'opaque'}
+    response = requests.post('http://{}/oauth/token'.format(url), headers=headers,
+                             params=payload,
+                             auth=(admin_id, admin_secret))
+    logger.info(f'Admin login returned {response.status_code}')
+    if response.status_code != 200:
+        exit(1)
+    resp_json = response.json()
+    access_token = resp_json.get('access_token')
+    
+    return access_token
+
+
+def update_info(token, url, authorities, authorized_grant_types, client_id, scope):
+    headers = {'Content-Type': 'application/json',
+               'Accept': 'application/json',
+               'Authorization': f'Bearer {token}'}
+
+    payload = {'client_id': client_id,
+               'authorized_grant_types': authorized_grant_types.split(", "),
+               'authorities': authorities.split(", "),
+               'scope': authorities.split(", "),
+               'resource_ids': 'oauth'}
+    
+    response = requests.post(f'http://{url}/oauth/clients/response_operations')
```

Comment:
Shouldn't this be a put rather than a post? (https://docs.cloudfoundry.org/api/uaa/version/74.3.0/index.html#update-6)